examples/dt/uni05epsilon/data-plane.md

```diff
@@ -0,0 +1,97 @@
+# Deploying the OpenStack dataplane
+
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been successfully deployed.
+
+
+## Initialize
+
+Switch to the "openstack" namespace
+
+```bash
+oc project openstack
+```
+
+Change to the Epsilon's directory
+
+```bash
+cd architecture/examples/dt/uni05epsilon
+```
+
+Modify the [values.yaml](values.yaml) with the following information
+
+- SSH keys to be used for accessing the deployed compute nodes.
+- SSH keys to be use for Nova migration.
+
+> All values must be in base64 encoded format.
+
+
+### Compute access
+
+1. Set `data['authorized']` with the value of all OpenStack Compute host SSH
+  keys.
+2. Set `data['private']` with the contents of the SSH private key to be used
+  for accessing the dataplane compute nodes.
+3. Set `data['public']` with the contents of the SSH public key used for
+  accessing the dataplane compute nodes.
+
+
+### Nova migration
+
+1. Set `data['nova']['migration']['ssh_keys']['private']` with the content of
+  the SSH private key to be used for potential future migration.
+2. Set `data['nova']['migration']['ssh_keys']['public']` with the content of
+  the SSH public key to be used for potential future migration.
+
+
+## CRs nodeset
+
+Generate the nodeset CRs.
+
+```bash
+kustomize build > edpm-nodeset.yaml
```

Comment:
Is it expected that this command produce CRs of kinds NetworkAttachmentDefinition, IPAddressPool, L2Advertisement, NetConfig?

Reply:
Hmm... So it seems those should be only under `control-plane/`? (I just compared with uni04delta)
So I must have made a mistake somewhere, but I am not so proficient with kustomizations yet, so if you can help me narrowing where I made mistake it would be great :pray:

Reply:
Ok, it appears due to `dt/uni05epsilon/edpm/nodeset/kustomization.yaml` specifies `octavia-network-attachment-definition.yaml` resource and I need `lib/networking/nad` component so kustomize is not complaining.
I can remove `lib/networking/metallb` and `lib/networking/netconfig`, then there is only `NetworkAttachmentDefinition` here.

But now I started to wonder I should keep the resources (`octavia-network-attachment-definition.yaml` and `cinder-volume-*-secrets.yaml`) under `dt/uni05epsilon/edpm/nodeset/` or rather directly `dt/uni05epsilon/`? (To have it in `control-plane/` only)

Reply:
Ok, I moved the resources to the control-plane-related kustomization only, the deployment went fine with that and in edpm-nodeset there is no more L2Advertisement,NetConfig,IPAddressPool,NetworkAttachmentDefinition stuff there :+1: